src/core/core.datasetController.js

```diff
@@ -591,7 +614,8 @@ export default class DatasetController {
 	getMinMax(scale, canStack) {
 		const me = this;
 		const meta = me._cachedMeta;
-		const _parsed = meta._parsed;
+		// Use filtered parsed elements if the exist, otherwise the full elements
```

Comment:
isn't the correct array always available in the `meta._parsed`?

Reply:
I'm storing the full array in `meta._parsedRaw` and the filtered one in `meta._parsed`

Reply:
if there is not filtered array, `meta._parsed === meta._parsedRaw`?